modules/tensor_mechanics/test/tests/volumetric_locking_verification/42_node.i

```diff
@@ -0,0 +1,105 @@
+# Test for volumetric locking correction
+
+# 2D cook's membrane problem with a trapezoid
+# that is fixed at one end and is sheared at
+# other end. Poisson's ratio is 0.4999.
+
+# Using Quad4 elements and no volumetric locking,
+# vertical displacement at top right corner is 3.78
+# due to locking.
+
+# Using Quad4 elements with volumetric locking, vertical
+# dispalcement at top right corner is 7.78.
+
+# Results match with Nakshatrala et al., Comp. Mech., 41, 2008.
+
+# Check volumetric locking correction documentation for
+# more details about this problem.
+
+[GlobalParams]
+  displacements = 'disp_x disp_y'
+  volumetric_locking_correction = true
+[]
+
+[Mesh]
+  file = 42_node_side.e
```

Comment:
@sveerara Do you have an accompanying journal file for this mesh? I would like to debug a new feature and this problem is simply too big. In general it can be very frustrating for future developers when folks check in Exodus meshes without an accompanying journal file.

Reply:
@lindsayad, I don't have the journal file for this exodus file but I am working on creating the journal file now. I will check that in today. Thanks for your comment.